starfish/image/_filter/__init__.py

```diff
@@ -1,6 +1,6 @@
 import argparse
 
-from starfish.image import ImageStack
+from starfish._stack import ImageStack
```

Comment:
Can you just do: from starfish import ImageStack?

Reply:
Yes. I'm not opinionated about how the paths should look. There are two options: 

1. Use the paths supported by __init__ in both the code and public API
2. Use the paths supported by __init__ only in the public API, and import from the files that modules are found in the codebase. 

I'm open to either. (1) produces uglier code that's slightly easier to follow than (2) because you don't have to open the __init__ module to figure out where the code lives.

Reply:
I don't have a strong feeling about (1) vs (2), but if you want to do (2), then I think we should just make _stack public.